src/App/root_reducer.js

```diff
@@ -0,0 +1,8 @@
+import {combineReducers} from 'redux';
+import login from '../Login/reducer';
```

Comment:
[We can do this](https://www.npmjs.com/package/app-module-path), but it has downsides. Notably, it becomes more confusing which modules are internal vs external dependencies. Let's get other opinions on this. @dennismphil?